daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/SBuiltin.scala

```diff
@@ -1583,7 +1583,7 @@ private[lf] object SBuiltin {
         machine: Machine,
     ): Unit = {
       val message = getSText(args, 0)
-      machine.traceLog.add(message, machine.lastLocation)
+      machine.traceLog.add(message, machine.lastLocation)(machine.loggingContext)
```

Comment:
Not sure whether this is the intent, but consider that here you are losing any context that has been accrued as part of the call stack that lead to the `execute` method call.

Reply:
This is somehow intended, not sure it is exactly the idea that @cocreature has that in mind. 
In this particular point, the Daml user is not interested by the interpretation stack trace, but  he should be able to identified the command that triggers this log. Note that in general, the interpretation stack trace, is pretty useless anyway. If the user what to have some stack information of its Daml program it has to active Daml stack tracing. 

Note: We could think adding this info here, when the stack tracing of the machine is on.

Reply:
Yes that’s intended. We want the context of the command submission so we get command ids, trigger ids, … but we don’t need additional context build up during command interpretation.